service-api/app/src/App/src/DataAccess/Repository/UserLpaActorMapInterface.php

```diff
@@ -20,15 +18,23 @@ interface UserLpaActorMapInterface
      * @param string $lpaActorToken
      * @param string $userId
      * @param string $siriusUid
-     * @param int $actorId
+     * @param string $actorId
```

Comment:
With the reversions to the parameter name changes, and given the confusion, it could be worth adding some parameter description text. i.e.

```suggestion
     * @param string $actorId The Sirius formated UID that is associated with an actor in the system
```